metadata_service/api/column.py

```diff
@@ -60,3 +62,35 @@ def get(self, table_uri: str, column_name: str) -> Union[tuple, int, None]:
 
         except Exception:
             return {'message': 'Internal server error!'}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+class ColumnBadgeAPI(Resource):
```

Comment:
it looks like it's mostly dashboard code that's using that capability -- so while it's certainly option to implement it, it's encouraged since safer and better error messages

Reply:
ignoring for now ~